nrf_security/sst_settings/Kconfig

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+config SST_SETTINGS
+	bool "Settings module for Secure Storage"
+	help
+	  Enable to get access for flashin operation based on Settings subsys.
```

Comment:
Not sure about this sentence. 
Maybe "Enable Secure Storage flash read/write operations using the Settings subsystem."